packages/gatsby-plugin-sharp/src/index.js

```diff
@@ -411,19 +420,33 @@ async function notMemoizedbase64({ file, args = {}, reporter }) {
     return null
   }
 
+  // Not all tranformer plugins are going to provide this parameter hence the fallback
+  if (cache) {
+    const cachedBase64 = await cache.get(base64CacheKey(file.internal.contentDigest, options))
+    if (cachedBase64) {
+      return {
```

Comment:
Could we just cache the entire object so we don't need to compute anything here?